x-pack/plugins/observability/dev_docs/slo.md

```diff
@@ -1,6 +1,69 @@
 # SLO
 
-Starting in 8.8, SLO is enabled by default.
+Starting in 8.8, SLO is enabled by default. SLO is GA since 8.12
+
+
+## Development & testing
+
+1. Data generation
+
+> [!TIP]
+> The following commands uses [kbn-data-forge](../../../../packages/kbn-data-forge/README.md) to generate some data for developping or testing SLOs
+
+Basic command to generate 7 days of data with a couple of services:
+```sh
+node x-pack/scripts/data_forge.js \
+	--events-per-cycle 100 \
+	--lookback now-7d  \
+	--dataset fake_stack \
+	--install-kibana-assets \
+	--kibana-url http://localhost:5601/kibana
+```
+
+Command to generate data for 7 days, including some ephemeral project ids. This is useful for working on the groupBy feature:
+```sh
+node x-pack/scripts/data_forge.js \
+	--events-per-cycle 50 \
+	--lookback now-7d \
+	--ephemeral-project-ids 10 \
+	--dataset fake_stack \
+	--install-kibana-assets \
+	--kibana-url http://localhost:5601/kibana
+```
+
+Get help with the data forge tool: `node x-pack/scripts/data_forge.js --help`
+
+2. Create SLOs
+
+> [!TIP]
+> Using the API is possible, but to prevent this document of becoming out of date, we refer to the [openAPI](../docs/openapi/slo/bundled.yaml) specification instead. 
+> Using the UI for developping/testing is the simpler approach.
+
+
+Start your kibana instance as usual, and open [kibana](http://localhost:5601/kibana/app/observability/slos). You might need to use the correct base path for your setup.
+On this page, you'll be able to create SLOs.
+
+
+> [!WARNING]
+> Wait for the data to be fully generated (it can take a few minutes to a couple of hours depending of the settings you use).
```

Comment:
Is there any message or indication when the script is finished?

I used the command that you shared and it took more than a few minutes, so I changed it to now-1d to see how long it would take. I am wondering if it makes sense to have a command that will be finished sooner and mention how we can add extra data by changing the time range in the command.

Reply:
I actually never looked at the cli logs 🙈 Always looked at the discover screen